irods/test/connection_test.py

```diff
@@ -236,6 +239,18 @@ def test_assigning_session_connection_timeout__issue_569(self):
             sess, old_timeout
         )
 
+    def test_legacy_auth_codepath__issue_499(self):
+        import irods.client_configuration as config
+        with config.loadlines(
+            entries=[dict(setting="legacy_auth.force_legacy_auth", value=True)]
+        ):
+            stream = io.StringIO()
+            logger = logging.getLogger("irods.connection")
+            with helpers.enableLogging( logger, logging.StreamHandler, (stream,), level_=logging.INFO):
+                with temp_setter(logger,"propagate",False):
+                    helpers.make_session().collections.get('/')
+        regex = re.compile('^.*Native auth.*(in legacy auth).*$',re.MULTILINE)
+        self.assertTrue(regex.search(stream.getvalue()))
```

Comment:
I am a little nervous about relying on logs in tests based on experiences in the main repo, but if this is standard practice and works consistently in PRC, then this is fine. Please resolve with impunity in the case that it is normal.

Reply:
Seems fine, then. Fine til proven not-fine. Resolving.